src/main/java/uk/gov/justice/hmpps/offenderevents/subscribe/SubscribeLocalstackConfig.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.hmpps.offenderevents.subscribe;
+
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class SubscribeLocalstackConfig {
+
+    @Bean
+    @ConditionalOnProperty(name = "sqs.provider", havingValue = "localstack", matchIfMissing = true)
```

Comment:
Does matchIfMissing mean we assume localstack if there is no sqs.provider?

Reply:
Yes, rightly or wrongly I copied the existing pattern in this project - although it really isn't needed 
I'll probably take out is subsequent PRs if I have the energy